test/lib/ansible_test/_internal/classification.py

```diff
@@ -668,6 +668,9 @@ def _classify_collection(self, path):  # type: (str) -> t.Optional[t.Dict[str, s
         if path.startswith('docs/'):
             return minimal
 
+        if path in ('README.md', 'CHANGELOG.rst', 'COPYING', 'LICENSE'):
```

Comment:
Should we classify all `*.md` and `*.rst` files at the root as needing only `minimal` instead of specific ones?

That's what we do for ansible-base:  https://github.com/ansible/ansible/blob/a51a6f4a259b45593c3f803737c6d5d847258a83/test/lib/ansible_test/_internal/classification.py#L891-L892

Reply:
That would also be fine for me. None of these files should be relevant for ansible-test.

(Not sure about `.in`, `.toml` and `.txt` though. The latter could be used for `requirements.txt` - though I'm not sure whether tests should actually use it. I guess they can though.)

Reply:
Returning `minimal` will result in sanity tests being executed as appropriate on the files. By classifying them we're telling `ansible-test` it doesn't need to fall back to testing everything due to lack of classification.

Tests shouldn't be using a `requirements.txt` in the root of the repository, and extra work is required to access files outside the current test target for integration tests, so it shouldn't happen accidentally.

Reply:
I've adjusted the PR in 7d6c3010968f289eb7d0c540e730096c5c24c1d2.